docs/contributing/managing-change.md

```diff
@@ -171,3 +171,11 @@ Keep the old name in the selector list, and mark it as deprecated.
   foo: bar;
 }
 ```
+
+### The `_warning` mixin and allowing users to suppress warnings
+
+In the above examples we've used `@include _warning(...)` instead of the native sass `@warn` at-rule. We do this instead of using `@warn` because it gives users the option to suppress deprecation warnings by interacting with the `$govuk-suppressed-warnings` map.
+
+The `_warning` mixin takes 2 parameters: a warning key (`$key`) and a warning message (`$message`), checks if `$key` isn't already included in `$govuk-suppressed-warnings` and if it isn't, outputs a warning via `@warn` using `$message` plus a notice to add the `$key` to `$govuk-suppressed-warnings` if they want to silence the warning.
+
+We allow users to do this because they don't always have the means to action deprecation warnings or upgrade their codebase beyond a specific version of GOV.UK Frontend eg: a legacy codebase that doesn't have the resource to upgrade to the latest breaking change where a deprecated feature will be removed. This feature allows those users to continue to operate their codebase without having to "tolerate" deprecation warnings in their testing.
```

Comment:
```suggestion
We make this option available for users because they can not always action deprecation warnings or upgrade their codebase beyond a specific version of GOV.UK Frontend. For example, a legacy codebase that does not have the resource to upgrade to the latest breaking change where a deprecated feature will be removed. This feature allows those users to continue to operate their codebase without having to repeatedly see non-actionable deprecation warnings in their testing.
```